### ISSUE TITLE
IA - site

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/network#41]
Update IA to detect any gaps since the changes we made in the structure during the workweek.

Simplified version updated (for staff presentation): 
![simplified-ia-v2](https://cloud.githubusercontent.com/assets/22157921/22451918/4b06e374-e727-11e6-81e7-80f27c2e0282.png)

Modify the full version with the latest updates

**Comment by xmatthewx** [comment]
Thanks @taisdesouzalessa for the update. A few suggestions to simplify for easier reading by stakeholders:

 - Condense the three offerings into a single "Offerings" item. These will change often.
 - The note below Join Us can be shortened to just "On all pages"
 - The yellow arrows below that come from the About page can be removed. A stakeholder just needs to see the summary of pages, not all the links.  
 - Chat link in footer is incorrect. Let's remove this item. We can add it back if and when we move to slack.
 - My gut says that News and Offerings should be on the right side, main menu on the left. I might be mistaken, but I think that would be more intuitive to understand the site at a glance.

Do we need to maintain a full version? It isn't a reference we've leaned on yet. Is it useful or extra work?

**Comment by taisdesouzalessa** [comment]
I think we can just maintain the simplified one. Some things changed since then - now we have an embedded "Thank You" - I'll update it an post it here.

**Comment by taisdesouzalessa** [comment]
Revised simplified IA:
<img width="924" alt="screen shot 2017-02-21 at 5 31 59 pm" src="https://cloud.githubusercontent.com/assets/22157921/23193030/a379c392-f85b-11e6-82e5-cdbb85d96042.png">


It is updated in our [Summary Deck](https://docs.google.com/presentation/d/1nLLNCyp1JtrOdiKlFYoBMOa_VqpXFrbxeYRI5PgwB0A/edit?usp=sharing)

**Comment by taisdesouzalessa** [comment]
In the footer links, we don't have "code of conduct". Should we add it?

**Comment by xmatthewx** [comment]
Where would Code of Conduct link? We might need to create a page for that on the site. Not a bad idea. Might call it Participation Guidelines like on Mozfest: https://mozillafestival.org/guidelines

Let's a file a separate ticket for this.

**Comment by xmatthewx** [comment]
Posted ticket here: https://github.com/mozilla/network/issues/87

**Comment by hannahkane** [comment]
Is this where we should finalize the navigation and labels? @xmatthewx - can you summarize what changes we need to make and whether there are any outstanding decisions to be made?

**Comment by xmatthewx** [comment]
We're tracking revisions to Nav here: https://github.com/mozilla/network/issues/106. We can update this later.

**Comment by taisdesouzalessa** [comment]
Closing this ticket as new changes happened in the IA of the page and we now we have a detailed view of the revised IA in #352
